kubernetes/deployment-production.tmpl

```diff
@@ -36,4 +36,29 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 80
-  type: NodePort
\ No newline at end of file
+  type: NodePort
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: zoomapper-production-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
```

Comment:
This can go, it's actually best not to use this unless we do explicit URL rewriting at the ingress (we don't here), see https://github.com/zooniverse/static/pull/176 for details (happy to explain the details but it's todo with path encoding / decoding and what path data makes it to the upstream target)
```suggestion
```